operator/main.go

```diff
@@ -96,6 +103,7 @@ func main() {
 		<-signals
 		gops.Close()
 		close(shutdownSignal)
+		contextCancelFunc()
```

Comment:
Related the the comment below, this package contextCancelFunc variable might be nil.